tox.ini

```diff
@@ -14,11 +14,13 @@ setenv =
 deps = -r{toxinidir}/test-requirements.txt
 commands = python -m unittest discover
 
+[testenv:linters]
+basepython = python3
+commands = flake8 {posargs}
+
 [testenv:pep8]
```

Comment:
I'd remove this entry point, and update the env_list on the top live to be linters, not pep8

Reply:
done, thx